docs/reference/commandline/history.md

```diff
@@ -80,7 +80,6 @@ The following example uses a template without headers and outputs the
 `ID` and `CreatedSince` entries separated by a colon for all images:
 
 ```bash
-{% raw %}
```

Comment:
these are no longer needed for the CLI reference, because they're converted to YAML, and don't go through the pre-processor there